examples/stack/sample-aws-stack-nb.json

```diff
@@ -5,11 +5,20 @@
       "id": "example-notebook",
       "service": "workspace",
       "properties": {
-        "source_path": "dev/notebook.scala",
-        "path": "/Shared/stack-cli-examples/dev/notebook",
+        "source_path": "project/notebook.scala",
+        "path": "/tmp/stack-cli-examples/workspace-example/notebook",
         "object_type": "NOTEBOOK"
       }
     },
+    {
+      "id": "example-dbfs",
```

Comment:
Let's remove this one, as it does not make sense to put notebook source codes into dbfs. 
(There is no story. One story is to put a jar in dbfs and then use that jar in jobs)